meilisearch-http/src/task.rs

```diff
@@ -342,7 +350,7 @@ impl From<Vec<TaskView>> for TaskListView {
 #[serde(rename_all = "camelCase")]
 pub struct SummarizedTaskView {
     uid: TaskId,
-    index_uid: String,
+    index_uid: Option<String>,
```

Comment:
Do we want to display a `null` here or do we prefer to remove the field from the payload by not serializing it? What does the spec says?

Reply:
the specs says `null`:
> The 202 Accepted (Summarized task response) and the related task object define the indexUid to null since dump are related to all indexes.